src/main/java/services/actions/ReturnAndSupplyHandler.java

```diff
@@ -0,0 +1,10 @@
+package services.actions;
+
+import db.Storage;
+
+public class ReturnAndSupplyHandler implements ActionHandler {
```

Comment:
```suggestion
public class IncreaseFruitHandler implements ActionHandler {
```
We may have more operations handled with this class